### DISCUSSION TITLE
Docker Container Password

### DISCUSSION THREAD

**Discussion by Outsmart2383** [tteck/Proxmox#2457] [Help Needed] [answered]
Sorry, I'm a complete n00b and this is probably super simple, but if I spin up a Docker LXC, and allow the default settings, it sets up automated login. But if I then want to access the container via SSH or WinSCP I get prompted for a user and password, but I don't know what these are, and if I enter blanks, access is denied. If I'm planning on accessing the container via anything other than Proxmox console do I need to _not_ use default settings, or am I missing something else? Thanks!

**Comment by Outsmart2383** [comment]
Okay, so steps I've tried so far:
- Added a password to root using: sudo passwd root
- Enabled SSH login via root with: nano /etc/ssh/ssh_config followed by changing PermitRootLogin to "yes", then systemctl restart ssh

Still getting access denied when trying to use WinSCP, albeit the above does allow me SSH into it using Putty.

**Reply by tteck** [reply, marked as answer]
https://github.com/tteck/Proxmox/discussions/324

**Reply by Outsmart2383** [reply]
Ahhh, so the only solution is to remove auto login? (I've done that and it's working.)

I had seen that as _a_ solution, but I was hoping I could keep auto login whilst also being able to manually login. Thanks for the quick response!
